### ISSUE TITLE
Crashing when i use most transition keys in menus

### ISSUE THREAD

**Issue by bibble24** [CleverRaven/Cataclysm-DDA#24142]
Game version: #7512

Operating system: win 10

Tiles or curses: using none

Mods active: none

### Expected behavior: 
Being able to use transition keys in menus.

### Actual behavior: 
Game crashes when I'm in a menu and try to use left key, right key, occasionally tab and occasionally enter.

### Steps to reproduce the behavior: 
Launch game, press one of the keys listed above.

**Comment by ZhilkinSerg** [comment]
Can you please backup and remove `config` folder and then test if issue is still reproduced?

**Comment by bibble24** [comment]
I tried removing the config but it still doesn't work. the same thing happens.

**Comment by SomeDeadGuy** [comment]
Try to delete everything in "sound" folder.

**Comment by bibble24** [comment]
that fixed it! thank you.

**Comment by ZhilkinSerg** [comment]
Did you install some kind of sound pack?

**Comment by bibble24** [comment]
No, the sound on my computer no longer works so i guess it crashed becuase it was trying to make sound when it couldn't it just broke it

**Comment by SomeDeadGuy** [comment]
Game will crash if you do not have audiocard, or something like that.

**Comment by ZhilkinSerg** [comment]
Hmm... We could probably try to fix it, but I believe it is an issue with SDL library in the first hand.

**Comment by ZhilkinSerg** [comment]
Reproduced it with disabled sound card with following errors in the logs:

```
20:12:36.828 SDL : src/sdltiles.cpp:459: Failed to open audio mixer, sound won't work: DirectSoundCreate8: No audio device found
20:12:37.565 SDL : src/sdltiles.cpp:2194: Failed to load audio file data/sound/Basic/menu_move.ogg: Audio device hasn't been opened
20:12:37.565 SDL : src/sdltiles.cpp:2194: Failed to load audio file data/sound/Basic/menu_error.ogg: Audio device hasn't been opened
```

Strange thing I had sound disabled in the options menu:

```json
  { "info": "If true, music and sound are enabled.", "default": "Default: True", "name": "SOUND_ENABLED", "value": "false"
```

**Comment by AMurkin** [comment]
Is there file named soundpack.txt.disabled in the one of the soundpack folders?

**Comment by alanbrady** [comment]
Root cause appears to be that after src/sdltiles.cpp:473 we fail to open the audio device but if it was built with SOUND=1 we still try to do some stuff on a non-existent audio device, even if it's set to `false` in the config.  I'm working out a solution, should have a PR up soon.
